src/libstore/path-info.hh

```diff
@@ -78,15 +77,11 @@ struct ValidPathInfo
 
     void sign(const Store & store, const SecretKey & secretKey);
 
+    std::optional<StorePathDescriptor> fullStorePathDescriptorOpt() const;
+
     /* Return true iff the path is verifiably content-addressed. */
     bool isContentAddressed(const Store & store) const;
 
-    /* Functions to view references + hasSelfReference as one set, mainly for
-       compatibility's sake. */
-    StorePathSet referencesPossiblyToSelf() const;
-    void insertReferencePossiblyToSelf(StorePath && ref);
-    void setReferencesPossiblyToSelf(StorePathSet && refs);
-
```

Comment:
These don't exist, and made it into this header by mistake (my fault long ago! As it so happens due to an earlier version of this PR.)